tests/suites/test_suite_psa_crypto_init.function

```diff
@@ -187,10 +204,19 @@ void init_deinit(int count)
     psa_status_t status;
     int i;
     for (i = 0; i < count; i++) {
+        mbedtls_test_set_step(2 * i);
         status = psa_crypto_init();
         PSA_ASSERT(status);
+        if (!check_stats()) {
+            goto exit;
```

Comment:
An `exit:` label is added automatically by the test generation, which is fine when we're using `TEST_EQUAL()` and friends because the label is entirely hidden below the test abstractions.

In this case it looks confusing because we're using `goto exit;` explicitly but there appears to be no exit label. Could you add an explicit `exit` label at the bottom of this function for consistency?